src/pudl/metadata/enums.py

```diff
@@ -1,97 +1,39 @@
 """Enumerations of valid field values."""
 
-US_STATES: dict[str, str] = {
-    "AK": "Alaska",
-    "AL": "Alabama",
-    "AR": "Arkansas",
-    "AZ": "Arizona",
-    "CA": "California",
-    "CO": "Colorado",
-    "CT": "Connecticut",
-    "DE": "Delaware",
-    "FL": "Florida",
-    "GA": "Georgia",
-    "HI": "Hawaii",
-    "IA": "Iowa",
-    "ID": "Idaho",
-    "IL": "Illinois",
-    "IN": "Indiana",
-    "KS": "Kansas",
-    "KY": "Kentucky",
-    "LA": "Louisiana",
-    "MA": "Massachusetts",
-    "MD": "Maryland",
-    "ME": "Maine",
-    "MI": "Michigan",
-    "MN": "Minnesota",
-    "MO": "Missouri",
-    "MS": "Mississippi",
-    "MT": "Montana",
-    "NC": "North Carolina",
-    "ND": "North Dakota",
-    "NE": "Nebraska",
-    "NH": "New Hampshire",
-    "NJ": "New Jersey",
-    "NM": "New Mexico",
-    "NV": "Nevada",
-    "NY": "New York",
-    "OH": "Ohio",
-    "OK": "Oklahoma",
-    "OR": "Oregon",
-    "PA": "Pennsylvania",
-    "RI": "Rhode Island",
-    "SC": "South Carolina",
-    "SD": "South Dakota",
-    "TN": "Tennessee",
-    "TX": "Texas",
-    "UT": "Utah",
-    "VA": "Virginia",
-    "VT": "Vermont",
-    "WA": "Washington",
-    "WI": "Wisconsin",
-    "WV": "West Virginia",
-    "WY": "Wyoming",
+from pudl.metadata.dfs import POLITICAL_SUBDIVISIONS
+
+COUNTRY_CODES_ISO3166: set[str] = set(POLITICAL_SUBDIVISIONS.country_code)
+SUBDIVISION_CODES_ISO3166: set[str] = set(POLITICAL_SUBDIVISIONS.subdivision_code)
+EPACEMS_STATES: set[str] = set(
+    POLITICAL_SUBDIVISIONS.loc[
+        POLITICAL_SUBDIVISIONS.is_epacems_state, "subdivision_code"
+    ]
+)
+DIVISION_CODES_US_CENSUS: set[str] = set(
+    POLITICAL_SUBDIVISIONS.division_code_us_census.dropna()
+)
+
+APPROXIMATE_TIMEZONES: dict[str, str] = {
```

Comment:
I put `STATE_TO_CENSUS_DIVISION` in for the fuel price imputation.  I think I created `STATE_ID_FIPS` while trying to get rid of the weird list-of-dicts that's being used in `pudl.analysis.state_demand` but ultimately that construct was too entangled with the rest of the code in the module and I didn't feel like spending the time to untangle it, so just reconstructed the list-of-dicts based on the `POLITICAL_SUBDIVISIONS` table.

I went back and forth on whether to define these right next to where they're being used, and ended up wanting to keep them all together since they're all being derived from the same resource, and to hopefully encourage them to get re-used if the same set / mapping is needed again, rather than having a proliferation of potentially slightly different extracts from the same table.

The one case where I kept the definition right next to where it's used is the bespoke list-of-dicts that Ethan created in the